modules/gapi/include/opencv2/gapi/core.hpp

```diff
@@ -34,6 +34,9 @@ namespace cv { namespace gapi {
  * Core module functionality.
  */
 namespace core {
+    using GResize = cv::gapi::imgproc::GResize;
+    using GResizeP = cv::gapi::imgproc::GResizeP;
```

Comment:
Why are GResize and GResizeP located here (in core)?

Reply:
That aliases for backward compatibility too